Source/OCMock/OCMockObject.m

```diff
@@ -95,7 +97,14 @@ - (instancetype)init
         [recorder setMockObject:self];
         return (id)[recorder init];
     }
-    
+
+	// Prevent initialization in production code from resetting mock state.
+	if (initialized) {
+		return self;
+	} else {
+		initialized = YES;
+	}
+
```

Comment:
Do we really need another variable or could we check whether, say, `stubs` is not `nil` as a sign that the mock has been initialised already?

Reply:
Done.